src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,106 @@
+package duke.ui;
+
+import duke.commands.Task;
+import duke.data.TaskList;
+
+/**
+ * Encapsulates the Ui component of Duke.
+ * This class deals with interactions with the user.
+ *
+ * @author: Jason Ng
+ * @version: Duke Level-9
+ */
+public class Ui {
+    /** The border to be printed for messages */
+    private static final String BORDER = "____________________________________________________________";
+    /** The introductory message to be printed for the start of the program */
+    private static final String INTRO = "Hello from\n"
+                                        +  " ____        _        \n"
+                                        + "|  _ \\ _   _| | _____ \n"
+                                        + "| | | | | | | |/ / _ \\\n"
+                                        + "| |_| | |_| |   <  __/\n"
+                                        + "|____/ \\__,_|_|\\_\\___|\n"
+                                        + "Im Duke\nWhat can I do for you?";
+
+    /**
+     * Prints the introductory message for Duke
+     */
+    public void PrintIntro() {
+        System.out.println(INTRO);
+    }
+
+    /**
+     * Prints the input message for Duke.
+     * Message is lined with the borders stored in the attribute of the class.
+     *
+     * @param message The message to be printed.
+     */
+    public void PrintMessage(String message) {
```

Comment:
I think method names should start with a lower case. Same for all the other methods in this class.

Reply:
Agreed!

https://se-education.org/guides/conventions/java/intermediate.html#naming